library.properties

```diff
@@ -1,10 +1,10 @@
 name=ArduinoRS485
-version=1.1.0
+version=1.1.1
```

Comment:
Be sure to remember that the `version` property has already been bumped when performing a release.

I say this because, in the case where some time passes between such a change and when the release is performed, it might be forgotten that a bump has already been done, and thus  the typical workflow of bumping immediately before a release would result in a potentially confusing version number skip (e.g., `1.1.0` -> `1.1.2`).